portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/site.vm

```diff
@@ -246,25 +246,31 @@
                         <div id="cropme">
                             <img style="maxWidth: 100%" />
                         </div>
-                        <div id="cropToolbar" class="btn-toolbar" style="display: none;">
-                            <div class="btn-group">
-                                <a class="profile-image-zoom-in btn btn-sm btn-default" href="javascript:void(0)"></a>
-                                <a class="profile-image-zoom-out btn btn-sm btn-default" href="javascript:void(0)"></a>
+                        <div id="cropToolbar">
+                            <div class="btn-toolbar">
+                                <div class="btn-group">
+                                    <button class="profile-image-zoom-in btn btn-sm btn-default" aria-label="${rloader.getString("pic_changer_zoom_in")}"></button>
+                                    <button class="profile-image-zoom-out btn btn-sm btn-default" aria-label="${rloader.getString("pic_changer_zoom_out")}"></button>
+                                </div>
+                                <div class="btn-group">
+                                    <button class="profile-image-rotate btn btn-sm btn-default" aria-label="${rloader.getString("pic_changer_rotate")}"></button>
+                                </div>
                             </div>
-                            <div class="btn-group">
-                                <a class="profile-image-pan-up btn btn-sm btn-default" href="javascript:void(0)"></a>
-                                <a class="profile-image-pan-down btn btn-sm btn-default" href="javascript:void(0)"></a>
-                                <a class="profile-image-pan-left btn btn-sm btn-default" href="javascript:void(0)"></a>
-                                <a class="profile-image-pan-right btn btn-sm btn-default" href="javascript:void(0)"></a>
-                            </div>
-                            <div class="btn-group">
-                                <a class="profile-image-rotate btn btn-sm btn-default" href="javascript:void(0)"></a>
+                            <div id="cropToolbar" class="btn-toolbar">
```

Comment:
id=cropToolbar is used twice

Reply:
https://sakaiproject.atlassian.net/browse/SAK-47135

Reply:
Of course